scipy/ndimage/src/_ni_label.pyx

```diff
@@ -302,15 +304,19 @@ cpdef _label(np.ndarray input,
         next_region = 2
 
         # Used for labeling single lines
```

Comment:
This comment should be moved further down, I think.

Reply:
I tried to move it closer to a sensible part of the control flow.